website/docs/feature-flag-tutorials/react/examples.md

```diff
@@ -0,0 +1,627 @@
+
+```js
+{
+    "eventType": "isEnabled",
+    "eventId": "b4455218-4a88-43aa-8712-b99186f46548",
+    "context": {
+        "sessionId": 386689528,
+        "appName": "cypress-realworld-app",
+        "environment": "default"
+    },
+    "enabled": true,
+    "featureName": "newNotificationsBadge",
+    "impressionData": true
+}
+```
+
+And the console.log for `getVariant` returns:
+
+```js
+{
+    "eventType": "getVariant",
+    "eventId": "c41aa58b-d2c7-45cf-b668-7267f465e01a",
+    "context": {
+        "sessionId": 386689528,
+        "appName": "cypress-realworld-app",
+        "environment": "default"
+    },
+    "enabled": true,
+    "featureName": "newNotificationsBadge",
+    "impressionData": true,
+    "variant": "announcementsIcon"
+}
+```
+
+You can find more information on `isEnabled` and `getVariant` in our [impression data docs](/reference/impression-data#impression-event-data).
+
+Now that the application is capturing impression events, you can configure the correct data fields and formatting to send to any analytics tool or data warehouse you use.
+
+
+### Application Metrics & Monitoring
+
+Under the [Metrics tab](http://localhost:4242/projects/default/features/newNotificationsBadge/metrics), you can see the general activity of the [Cypress Real World App from our React tutorial](/feature-flag-tutorials/react) in the development environment over different periods of time. If the app had a production environment enabled, we would also be able to view the amount of exposure and requests the app is receiving over time.
+
+
+![Metrics view image](/img/react-ex-metrics.png)
+
+
+Our metrics are great for understanding user traffic. You can get a better sense of:
+
+
+* What time(s) of the day or week are requests the highest?
+* Which feature flags are the most popular?
+
+
+Another use case for reviewing metrics is verifying that the right users are being exposed to your feature based on how you’ve configured your strategies and/or variants.
+
+Take a look at our [Metrics API documentation](/reference/api/unleash/metrics) to understand how it works from a code perspective.
+
+
+## Feature Flag Audit Logs in React
+
+Because a Feature Flag service controls the way an application behaves in production, it can be highly important to have visibility into when changes have been made and by whom. This is especially true in highly regulated environments, such as health care, insurance, banking, and others. In these cases (or similar), you might find audit logging useful for:
+
+
+1. Organizational compliance
+2. Change control
+
+Fortunately, this is straightforward in Unleash Enterprise.
+
+Unleash provides the data to log any change that has happened over time, at the flag level from a global level. In conjunction with Unleash, tools like [Splunk](https://www.splunk.com/) can help you combine logs and run advanced queries against them. Logs are useful for downstream data warehouses or data lakes.
```

Comment:
> at the flag level from a global level

Is this missing something? Are we talking about seeing changes from the flag level to the global level?